lib/Models/BaseMaps/defaultBaseMaps.ts

```diff
@@ -94,5 +94,19 @@ export function defaultBaseMaps(terria: Terria): BaseMapJson[] {
     contrastColor: "#000000"
   });
 
+  baseMaps.push({
+    item: {
+      id: "basemap-openstreetmap",
+      name: "OpenStreetMap",
+      type: "open-street-map",
+      url: "https://tile.openstreetmap.org/",
+      attribution:
+        "© <a href='https://www.openstreetmap.org/copyright'>OpenStreetMap contributors</a>",
+      opacity: 1.0
+    },
+    image: "build/TerriaJS/images/openstreetmap.png",
```

Comment:
Can this file be used in `test/Models/BaseMaps/BaseMapsModelSpec.ts` and `test/Models/TerriaSpec.ts` so `dark-matter.png` that belonged to one of the removed maps can be removed?

Reply:
Thanks. I have removed references to these images from the specs for now.
We are not testing anything about the images at this point.